code/game/objects/items/devices/radio/headset.dm

```diff
@@ -248,6 +248,13 @@
 	item_state = "headset"
 	ks2type = /obj/item/encryptionkey/heads/arkrep
 
+/obj/item/radio/headset/heads/arkinstructor
+	name = "ark soft instructor's headset"
```

Comment:
Non capitalised

Reply:
Yeah I actually fixed the ark soft rep's capitalization instead -- no headsets are capitalized in the codebase.